pkgs/games/marble-marcher-ce/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, fetchFromGitHub
+, sfml
+, anttweakbar
+, glm
+, eigen
+, glew
+, cmake
+}:
+stdenv.mkDerivation rec {
+  pname = "marble-marcher-ce";
+  version = "1.4.5";
+
+  src = fetchFromGitHub {
+    owner = "WAUthethird";
+    repo = "Marble-Marcher-Community-Edition";
+    rev = version;
+    hash = "sha256-m5i/Q4k5S4wcojHqMYS7e1W/Ph7q/95j3oOK2xbrHSk=";
+  };
+
+  buildInputs = [ sfml anttweakbar glm eigen glew ];
+  nativeBuildInputs = [ cmake makeWrapper copyDesktopItems ];
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  prePatch = ''
+    # the path /home/MMCE is always added to DESTDIR
+    # we change this to a more sensible path
+    # see https://github.com/WAUthethird/Marble-Marcher-Community-Edition/issues/23
+    substituteInPlace CMakeLists.txt \
+      --replace '/home/MMCE' '/share/MMCE'
+  '';
+
+  postInstall = ''
+    mkdir $out/bin
+    mkdir -p $out/share/icons/
+    # The executable has to be run from the same directory the assets are in
+    makeWrapper $out/share/MMCE/MarbleMarcher $out/bin/${pname} --chdir $out/share/MMCE
+    ln -s $out/share/MMCE/images/MarbleMarcher.png $out/share/icons/${pname}.png
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      exec = pname;
+      icon = pname;
+      desktopName = pname;
```

Comment:
It's usually best to avoid this kind of dependency on `pname`. We don't necessarily want binary names, desktop names etc to change if we decide to change `pname`.

Reply:
I think it's fine. I don't think I've ever seen a pname change in a way that would confuse a user.